src/views/docs/en/reference/configuration/function-config.md

```diff
@@ -37,14 +37,14 @@ views false
 
 Configure the deployed function with [the `@aws` pragma](../project-manifest/aws) and the following properties:
 
-- [`runtime`](#runtime) - string, Lambda runtime or alias: `nodejs14.x` (default), `python3.7`, `dotnetcore3.1`, `node`, `py`, `.net`, etc.
+- [`runtime`](#runtime) - string, Lambda runtime or alias: `nodejs16.x` (default), `python3.7`, `dotnetcore3.1`, `node`, `py`, `.net`, etc.
 - [`memory`](#memory) - number, between `128` and `3008` MB in 64 MB increments.
 - [`timeout`](#timeout) - number, in seconds (max `900`)
 - [`concurrency`](#concurrency) - number, `0` to AWS account maximum (if not present, concurrency is unthrottled)
 - [`provisionedConcurrency`](#provisionedconcurrency) - number, `1` to AWS account maximum (disabled by default)
 - [`layers`](#layers) - Up to 5 Lambda layer ARNs; **must be in the same region as deployed**
 - [`policies`](#policies) - Configure [AWS SAM policy templates](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-policy-templates.html)
-- [`architecture`](#architecture) - [AWS Architecture](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html) for the function: `x86_64` (default) or `arm64`
+- [`architecture`](#architecture) - [AWS Architecture](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html) for the function: `arm64` (default) or `x86_64`
```

Comment:
`arm64` is not the default (yet), see: https://github.com/architect/architect/issues/1304

Reply:
Before making these changes, I did a clean install of latest version of Architect, ran `init`, then `deploy` and observed what the deployed Lambda function was configured with:

<img width="743" alt="Screenshot 2023-08-14 at 21 39 05" src="https://github.com/architect/arc.codes/assets/113305/26f0cce4-b245-44b7-bd34-f6002138556f">

Reply:
Ah, wait, is this a difference between a "default" (as in, what Arc will do if it can't find a configure value) and the out-the-box configuration example? If so, is that a bit confusing for anyone other than me (quite possibly no, it's just me!)

Reply:
Yeah, there's our "recommended" defaults that we have in our `init` templates, and then there's **actual** defaults (ie what you get when you do not specify a runtime). I'm referring to the latter in this case!